### ISSUE TITLE
Learning rate option in Activait

### ISSUE THREAD

**Issue by sjcrs** [radiasoft/sirepo#5818]
A feature request from the IPAC workshop:

A user would like to have some control over learning rate hyper-parameter within the regression neural network window. 

Custom learning rate scheduling would also be a nice-to-have (see "snapshot ensembling")
